pkg/api/common/versions.go

```diff
@@ -146,7 +146,7 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.15.2":         false,
 	"1.15.3":         false,
 	"1.15.4":         true,
-	"1.15.5":         false, // disabled because of https://github.com/Azure/aks-engine/issues/2312
+	"1.15.5":         true, // disabled because of https://github.com/Azure/aks-engine/issues/2312, reenable because we patched it already.
```

Comment:
actually it's a little tricky that the private patch is here: https://github.com/Azure/kubernetes/releases/tag/v1.15.5-hotfix.20191113
not sure whether aks-engine could adopt this.

Reply:
this is for aks consumption only. we just need to make sure images in the below repo are updated with the patch.
	KubernetesImageBaseWithGlobalACR = "aksrepos.azurecr.io/mirror/"
	KubernetesImageBaseWithGlobalMCR = "mcr.microsoft.com/oss/kubernetes/"

Reply:
Does the *_f0.0.2 patched images include the required change from Andy?

defaultValue: "v1.12.8_v0.0.5,v1.13.10_v0.0.5,v1.13.11_v0.0.5,v1.13.12_f0.0.2,v1.14.6_v0.0.5,v1.14.7_v0.0.5,v1.14.8_f0.0.2,v1.15.3_v0.0.5,v1.15.4_v0.0.5,v1.15.5_f0.0.2"

Reply:
@xuto2 yes, the f0.0.2 contains the fix from Andy.

@andyzhangx this is only for aks release.